### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by usergreen**

### CODE REVIEW

File: content/ja/mate-desktop/applications/engrampa.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+{{< columns >}}
+
+Engrampaは、MATE Desktop Environment の公式なアーカイブマネージャです。tar や zip などのファイル圧縮ユーティリティのグラフィカルなフロントエンドです。これは File Roller のフォークです。
+
+Engrampa では、書庫（アーカイブ）からのファイル作成、変更および展開など最も一般的な操作をサポートします。また、書庫の内容を一覧表示したり、書庫に含まれているファイルを開いたりすることもできます。
+
+    <--->
+
+[![](../images/engrampa-window.png)](../images/engrampa-window.png)
```
Comment (mbkma) [inline content/ja/mate-desktop/applications/engrampa.md:23]: I moved the images to the static dir (#28). Please replace this with  ``

**Review by mbkma — COMMENTED** [review]
Please squash the commits into one single commit.

File: content/ja/mate-desktop/applications/engrampa.md (hunk 1 above)
Comment (raveit65) [reply content/ja/mate-desktop/applications/engrampa.md:23]: Seems that concern is open?

Comment (raveit65) [reply content/ja/mate-desktop/applications/engrampa.md:23]: @mbkma Maybe we merge that and fix the link after?

**Comment by raveit65** [comment]
I used your commits in another PR with the fixed link.
https://github.com/mate-desktop/mate-wiki/pull/35
Just merged.....

**Pull request closed without merging**